Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp

```diff
@@ -107,7 +107,7 @@ void ApplicationManifestParser::logDeveloperWarning(const String& message)
         m_document->addConsoleMessage(makeUnique<Inspector::ConsoleMessage>(JSC::MessageSource::Other, JSC::MessageType::Log, JSC::MessageLevel::Warning, makeString("Parsing application manifest "_s, m_manifestURL.string(), ": "_s, message)));
 }
 
-URL ApplicationManifestParser::parseStartURL(const JSON::Object& manifest, const URL& documentURL)
+const URL ApplicationManifestParser::parseStartURL(const JSON::Object& manifest, const URL& documentURL)
```

Comment:
No, we don't want that. If you want to make the function const, then const goes at the end of the declaration, not the beginning. In this patch, you made the return value const, which is not useful.

Reply:
oh, lol. C++ is fun :)